src/org/zaproxy/zap/extension/fuzz/payloads/generator/NumberPayloadGenerator.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2015 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ package org.zaproxy.zap.extension.fuzz.payloads.generator;
+
+import org.apache.log4j.Logger;
+import org.zaproxy.zap.extension.fuzz.payloads.DefaultPayload;
+import org.zaproxy.zap.utils.ResettableAutoCloseableIterator;
+
+/**
+ *
+ * @author steven
+ */
+public class NumberPayloadGenerator implements StringPayloadGenerator, ResettableAutoCloseableIterator<DefaultPayload> {
+
+    private static final Logger LOGGER = Logger.getLogger(NumberPayloadGenerator.class);
+    private long fromNo, toNo, increments,pos;
+
+    public NumberPayloadGenerator(long fromNo, long toNo, long inc) {
```

Comment:
The name of the variable "inc" should be normalised, in some places is "step" in others "inc" or "increments".